go.mod

```diff
@@ -62,6 +61,7 @@ require (
 	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/dvsekhvalnov/jose2go v0.0.0-20200901110807-248326c1351b // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/evmos/ethereum-ledger-go v0.0.3 // indirect
```

Comment:
why are there 2 evmos and ethereum ledger?

Reply:
Ethereum Ledger Go provides a general purpose Ethereum Ledger interface, based off of Go Ethereum's implementation, and Evmos Ledger Go uses Ethereum Ledger Go.